src/Pim/Component/Catalog/Builder/ProductTemplateBuilder.php

```diff
@@ -64,55 +42,42 @@ public function createProductTemplate()
     /**
      * {@inheritdoc}
      */
-    public function addAttributes(ProductTemplateInterface $template, array $attributes, $locale)
+    public function addAttributes(ProductTemplateInterface $template, array $attributes)
```

Comment:
@SamirBoulil not for me
I think to makes sense to have this method
the strange thing imo is the fact we don't use it